### ISSUE TITLE
XMPPError is not defined in XEP-0050

### ISSUE THREAD

**Issue by coffeeowl** [fritzy/SleekXMPP#346]
Hello, suddenly, out of nowhere this problem has popped up:

> File "/usr/local/lib/python2.7/dist-packages/sleekxmpp/plugins/xep_0050/adhoc.py", line 286, in >_handle_command_next
>          raise XMPPError('item-not-found')
>          NameError: global name 'XMPPError' is not defined

I can't find XMPPError being imported anywhere in adhoc.py

**Comment by mathieui** [comment]
Yes, this is an issue I found some days ago. You can import XMPPError instead of IqError in the imports at the top of the file.
